pig-tests/type-domains/sample-universe.ion

```diff
@@ -117,79 +116,80 @@
 (define partiql_basic
   (domain
     (sum projection
-      (project_list (* project_item 1))
-      (project_value expr))
+      (product project_list (items (* project_item 1)))
+      (product project_value (value expr)))
 
     (sum project_item
-      (project_all)
-      (project_expr expr (? symbol)))
+      (product project_all)
+      (product project_expr (value expr) (as_alias (? symbol))))
 
     (sum join_type
-      (inner)
-      (left)
-      (right)
-      (outer))
+      (product inner)
+      (product left)
+      (product right)
+      (product outer))
 
     (sum from_source
-      (scan expr (? symbol) (? symbol) (? symbol))
-      (join join_type from_source from_source (? expr)))
+      (product scan (expr expr) (as_alias (? symbol)) (at_alias (? symbol)) (by_alias (? symbol)))
+      (product join (type join_type) (left from_source) (right from_source) (predicate (? expr))))
 
-    (product expr_pair expr expr)
+    (product expr_pair (first expr) (second expr))
 
-    (product group_by_item expr (? symbol))
+    (product group_by_item (value expr) (as_alias (? symbol)))
 
     (product group_by_list
-      (* group_by_item 1))
+      (items (* group_by_item 1)))
 
     (product group_by
-      group_by_list
-      (? symbol))  // group as alias
+      (items group_by_list)
+      (group_as_alias (? symbol)))  // group as alias
 
     (sum case_sensitivity
-      (case_sensitive)
-      (case_insensitive))
+      (product case_sensitive)
+      (product case_insensitive))
 
     (sum scope_qualifier
-      (unqualified)
-      (qualified))
+      (product unqualified)
+      (product qualified))
 
     (sum set_quantifier
-      (all)
-      (distinct))
+      (product all)
+      (product distinct))
 
     (sum path_element
-      (path_expr expr)
-      (path_wildcard)
-      (path_unpivot))
+      (product path_expr (expr expr))
+      (product path_wildcard)
+      (product path_unpivot))
 
     (sum expr
       // Basic Expressions
-      (lit ion)
-      (id symbol case_sensitivity scope_qualifier)
-      (parameter int)
-      (not expr)
-      (plus (* expr 2))
-      (minus (* expr 2))
-      (times (* expr 2))
-      (divide (* expr 2))
-      (modulo (* expr 2))
-      (concat (* expr 2))
-      (like expr expr expr)
-      (between expr expr expr)
-      (path expr (* path_element 1))
-      (call symbol (* expr 1))
-      (call_agg symbol set_quantifier expr)
+      (product lit (value ion))
+      (product id (name symbol) (case case_sensitivity) (scope_qualifier scope_qualifier))
+      (product parameter (index int))
+      (product not (expr expr))
+      (product plus (operands (* expr 2)))
+      (product minus (operands (* expr 2)))
+      (product times (operands (* expr 2)))
+      (product divide (operands (* expr 2)))
+      (product modulo (operands (* expr 2)))
+      (product concat (operands (* expr 2)))
+
+      (product like (left expr) (right expr) (escape expr))
```

Comment:
nit: 
```suggestion
      (product like (value expr) (pattern expr) (escape expr))
```
* `left` -> `value`
* `right` -> `pattern`

Reply:
You know, I swear I was thinking "value" and "pattern" but my fingers typed "left" and "right"